app/views/tweeets/_feed.html.erb

```diff
@@ -0,0 +1,51 @@
+<div class="column is-half">
+  <% if user_signed_in? %>
```

Comment:
Such logic must be handled into **view helpers**

Reply:
> Such logic must be handled into **view helpers**
Can you give me an example for the if statement for this case please?

Reply:
Here it is. It is in  one of the articles provided in Where should I put my code links.
![image](https://user-images.githubusercontent.com/44072711/88323563-16eadd00-cd2b-11ea-826f-a8eb889e17cf.png)

Reply:
Thanks for the tips, i didn't find it in the link, but thanks anyway.
Have a nice day